src/Package/Impl/DataInspect/DataImport/ImportDataWindow.xaml.cs

```diff
@@ -287,7 +270,7 @@ class DataFramePreviewRowItem {
             await TaskUtilities.SwitchToBackgroundThread();
 
             Encoding encoding = Encoding.GetEncoding(codePage);
-            _tempFilePath = Path.GetTempFileName();
+            _utf8FilePath = Path.Combine(Path.GetTempPath(), Path.GetFileName(file) + ".utf8");
```

Comment:
Ok

---
In reply to: [60455799](https://github.com/Microsoft/RTVS/pull/1651#discussion_r60455799) [](ancestors = 60455799)